### PR TITLE
Fix notice and wrong Regex

### PR DESCRIPTION
**Pull request by timotei-litespeed**
Found after looking for this issue: https://wordpress.org/support/topic/php-deprecated-notice-14/

### CODE REVIEW

File: src/cdn.cls.php [hunk 1]
```diff
@@ -306,8 +306,12 @@ private function _replace_inline_css()
 		 * @see  #685485
 		 * @since 3.0
 		 */
-		preg_match_all('/url\((?![\'"]?data)[\'"]?([^\)\'"\\\]+)[\'"]?\)/i', $this->content, $matches);
+		preg_match_all('/url\((?![\'"]?data)[\'"]?(.*?)[\'"]?\)/i', $this->content, $matches);
 		foreach ($matches[1] as $k => $url) {
+			if ($url == null) {
```
Comment (hi-hai) [inline src/cdn.cls.php:311]: Should we just use `if (!$url)` as empty case should be bypassed as well??

Comment (timotei-litespeed) [reply src/cdn.cls.php]: Yes, added the changed

File: src/cdn.cls.php [hunk 2]
```diff
@@ -306,8 +306,12 @@ private function _replace_inline_css()
 		 * @see  #685485
 		 * @since 3.0
 		 */
-		preg_match_all('/url\((?![\'"]?data)[\'"]?([^\)\'"\\\]+)[\'"]?\)/i', $this->content, $matches);
+		preg_match_all('/url\((?![\'"]?data)[\'"]?(.*?)[\'"]?\)/i', $this->content, $matches);
```
Comment (hi-hai) [inline src/cdn.cls.php:309]: Should we change `(.*?)` to `(.+?)`? @timotei-litespeed

File: src/cdn.cls.php [hunk 3]
```diff
@@ -306,8 +306,12 @@ private function _replace_inline_css()
 		 * @see  #685485
 		 * @since 3.0
 		 */
-		preg_match_all('/url\((?![\'"]?data)[\'"]?([^\)\'"\\\]+)[\'"]?\)/i', $this->content, $matches);
+		preg_match_all('/url\((?![\'"]?data)[\'"]?(.+?)[\'"]?\)/i', $this->content, $matches);
 		foreach ($matches[1] as $k => $url) {
+			if (!$url) {
```
Comment (hi-hai) [inline src/cdn.cls.php:311]: As its used + now, is this condition redundant? @timotei-litespeed

Comment (timotei-litespeed) [reply src/cdn.cls.php]: removed

**Pull request merged**